modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/side_navigation.es.js

```diff
@@ -427,6 +447,206 @@ SideNavigation.prototype = {
 		}
 	},
 
+	_onClosed() {
+		const instance = this;
+		const options = instance.options;
+
+		const container = document.querySelector(options.container);
+
+		if (!instance._handleClosed) {
+			instance._handleClosed = () => {
+				const type = !instance._isDesktop()
+					? options.typeMobile
+					: options.type;
+
+				if (type === 'relative' && hasClass(container, 'open')) {
+					removeClass(container, 'sidenav-transition');
+				}
+			};
+
+			document.addEventListener(
+				'closed.lexicon.sidenav',
+				instance._handleClosed
+			);
+		}
+	},
+
+	_onClosedStart() {
+		const instance = this;
+		const options = instance.options;
+
+		const container = document.querySelector(options.container);
+		const content = document.querySelector(options.content);
+
+		if (!instance._handleClosedStart) {
+			instance._handleClosedStart = () => {
+				const type = !instance._isDesktop()
+					? options.typeMobile
+					: options.type;
+
+				if (
+					type === 'relative' &&
+					hasClass(container, 'open') &&
+					content.closest('.page-maximized')
+				) {
+					let contentMargin =
+						document.body.scrollWidth -
+						content.getBoundingClientRect().right;
+
+					let paddingRight = options.gutter + options.width;
+
+					const contentMaxWidth =
+						getComputedStyle(content).maxWidth ||
+						getComputedStyle(content).width;
+
+					if (contentMaxWidth.match(/px$/g)) {
```

Comment:
Although we are pretty slack about doing this consistently, `match()` is best used when you actually want to do something with the match. In this case, you are not even looking at it (beyond the truthiness evaluation implied by the `if`), so a better fit would be `exec`:

```js
if (/px$/.test(contentMaxWidth)) {
```

(Note that you don't need the `g` flag, either way.)